test/common/fns/randomVal.test.js

```diff
@@ -0,0 +1,83 @@
+import randomVal from '../../../common/script/fns/randomVal';
```

Comment:
It's correct to rename this file to `randomVal.test.js`, but to make reviewing the changes easier, let's leave it as is.